### DISCUSSION TITLE
Can examples be added to the PREWHERE documenation? https://github.com/ClickHouse/ClickHouse/blob/master/docs/en/sql-reference/statements/select/prewhere.md

### DISCUSSION THREAD

**Discussion by aaronstrat** [ClickHouse/ClickHouse#46414] [Ideas]
Examples would help me better understand the content of this page. https://github.com/ClickHouse/ClickHouse/blob/master/docs/en/sql-reference/statements/select/prewhere.md

**Comment by den-crane** [comment]
https://github.com/ClickHouse/ClickHouse/pull/46415/files

**Reply by aaronstrat** [reply]
Thanks!
